assets/js/modules/analytics-4/datastore/properties.js

```diff
@@ -490,6 +498,53 @@ const baseActions = {
 			.setGoogleTagContainerID( googleTagContainerID );
 		registry.dispatch( MODULES_ANALYTICS_4 ).setGoogleTagID( googleTagID );
 	},
+
+	/**
+	 * Syncs Google Tag settings.
+	 *
+	 * @since n.e.x.t
+	 */
+	*syncGoogleTagSettings() {
+		const { select, dispatch, __experimentalResolveSelect } =
+			yield Data.commonActions.getRegistry();
+
+		// Wait for modules to be available before selecting.
+		yield Data.commonActions.await(
+			__experimentalResolveSelect( CORE_MODULES ).getModules()
+		);
+
+		const { isModuleConnected } = select( CORE_MODULES );
```

Comment:
Is there a functional reason this line is between the two `yield` statements?

If not, I'd suggest moving it after them, and closer to where the selector is used, for a marginal improvement in organisation.

Reply:
I agree. However, I have re-structured the selector as a part of [this commit](https://github.com/google/site-kit-wp/pull/6536/commits/5d61a1f6ddbf2b58c033183dd5741813522aa964) to return early to prevent continuing with selections unnecessarily. Does this look good to you?

Reply:
Thanks, yes that looks much better, nice one :)